sqlitecluster/SQLite.cpp

```diff
@@ -256,7 +256,8 @@ int SQLite::_walHookCallback(void* sqliteObject, sqlite3* db, const char* name,
 
 void SQLite::_sqliteLogCallback(void* pArg, int iErrCode, const char* zMsg) {
     _mostRecentSQLiteErrorLog = "{SQLITE} Code: "s + to_string(iErrCode) + ", Message: "s + zMsg;
-    SSYSLOG(LOG_INFO, "[info] " << _mostRecentSQLiteErrorLog);
+    SRedactSensitiveValues(_mostRecentSQLiteErrorLog);
```

Comment:
I think that this should just be part of the previous line: `SRedactSensitiveValues(zMsg)`

Edit: OK, I see why it isn't, the argument is modified in place. We can eave it as-is.